### PR TITLE
chore: upgrade and rebuild ApollosStorybook

### PR DESCRIPTION
**Pull request by conrad-vanl**
upgrades storybook

Screen Shot 2022-01-28 at 11 30 14 PM

### CODE REVIEW

File: ApollosStorybook/Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
+ruby '~> 2.7.4'
```
Comment (redreceipt) [inline ApollosStorybook/Gemfile:4]: is there a reason we need to lock the ruby version? I use 2.6 and it never gives me problems with our apps. we don't lock the version on the template

Comment (conrad-vanl) [reply ApollosStorybook/Gemfile:4]: This is all just what's default in react-native unit. I can remove.

Comment (redreceipt) [reply ApollosStorybook/Gemfile:4]: Oh interesting. I kinda like that then. Maybe we should do it in the template too to avoid other RN problems?

Comment (conrad-vanl) [reply ApollosStorybook/Gemfile:4]: Sure. I just started with a fresh react-native init in this branch. After the fact, I thought I should've used our CLI to generate an Apollos app and add storybook to it 🙃 but already had this done

**Comment by redreceipt** [comment]
got this

Screen Shot 2022-01-29 at 1 32 51 PM

**Comment by redreceipt** [comment]
also this
Screen Shot 2022-01-29 at 1 34 53 PM

**Comment by vinnyjth** [comment]
@conrad-vanl did you want to target `next` or `master` here?

**Comment by vinnyjth** [comment]
Same issue as @redreceipt (and same diff)

Screen Shot 2022-01-31 at 9 23 12 AM

**Comment by conrad-vanl** [comment]
yeah, I started seeing this too...not sure why. See if this last commit fixes for y'all and I can better remove/look into it.

**Comment by conrad-vanl** [comment]
> also this

@redreceipt I'm not sure what I'm looking at here

**Comment by conrad-vanl** [comment]
> @conrad-vanl did you want to target next or master here?

@vinnyjth what's he point of `next` again? 😅  this doesn't depend on anything in `next`

**Comment by vinnyjth** [comment]
> > also this
> 
> @redreceipt I'm not sure what I'm looking at here

I'm seeing that too. It's the result of `git diff` after running `yarn storybook`

**Comment by vinnyjth** [comment]
> > @conrad-vanl did you want to target next or master here?
> 
> @vinnyjth what's he point of `next` again? 😅 this doesn't depend on anything in `next`

It's the `cluster` version of our app code / packages :) If you merge into `master`, it will also end up in `next` eventually :)

**Comment by vinnyjth** [comment]
Everything is working great w/ your recent changes @conrad-vanl :) Do you want me to commit the diff that @redreceipt and I are seeing?

**Comment by conrad-vanl** [comment]
@vinnyjth why would the frontend be considered with cluster or not :)

**Comment by conrad-vanl** [comment]
> Everything is working great w/ your recent changes @conrad-vanl :) Do you want me to commit the diff that @redreceipt and I are seeing?

Sure.

**Comment by vinnyjth** [comment]
@conrad-vanl Because `ui-authentication` doesn't work on non-cluster apps and the app itself has to depend on either `ui-auth` or `ui-authentication`

**Comment by vinnyjth** [comment]
https://3.basecamp.com/3926363/buckets/6947926/messages/4517838969

**Comment by conrad-vanl** [comment]
> I'm seeing that too. It's the result of git diff after running yarn storybook
@vinnyjth is your ruby not native arm64? those diffs makes sense coming from @redreceipt, but I'm intrigued why you and I aren't seeing the same thing.,

**Comment by vinnyjth** [comment]
```
⏲/differential-projects/apollos/apollos-apps [09:52] (storybook-upgrade🍅)
⇝ ruby --version
ruby 2.7.4p191 (2021-07-07 revision a21a3b7d23) [arm64-darwin21]
⇝ rvm --version
rvm 1.29.12 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
⇝ bundler --version
Bundler version 2.1.4
```

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**